flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/shuffle/TestAggregatedStatisticsTracker.java

```diff
@@ -246,7 +254,14 @@ public void receiveCompletedStatisticsEvent(StatisticsType type) {
                   CHAR_KEYS.get("a"), 3L,
                   CHAR_KEYS.get("b"), 3L));
     } else {
-      assertThat(completedStatistics.rangeBounds()).containsExactly(CHAR_KEYS.get("a"));
+      assertThat(completedStatistics.keySamples())
+          .containsExactly(
+              CHAR_KEYS.get("a"),
+              CHAR_KEYS.get("b"),
+              CHAR_KEYS.get("b"),
+              CHAR_KEYS.get("a"),
+              CHAR_KEYS.get("a"),
```

Comment:
So the key samples are not ordered, just randomly selected samples from the incoming records?

Reply:
I have expected a bit more (based on my tests with sketches over long values), but having arbitrary keys probably doesn't allow better approximation

Reply:
Sketch returns the array that captures the reservoir samples. before array is full, samples were added to the array in order.